trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/CatalogRegister.java

```diff
@@ -62,17 +64,31 @@ private void checkTrinoSpiVersion(ConnectorContext context) {
     this.trinoVersion = context.getSpiVersion();
 
     int version = Integer.parseInt(context.getSpiVersion());
-    if (version < MIN_TRINO_SPI_VERSION) {
+    if (version < MIN_SUPPORT_TRINO_SPI_VERSION || version > MAX_SUPPORT_TRINO_SPI_VERSION) {
       String errmsg =
           String.format(
-              "Unsupported Trino-%s version. min support version is Trino-%d",
-              trinoVersion, MIN_TRINO_SPI_VERSION);
+              "Unsupported Trino-%s version. The Supported version for the Gravitino-Trino-connector from Trino-%d to Trino-%d",
+              trinoVersion, MIN_SUPPORT_TRINO_SPI_VERSION, MAX_SUPPORT_TRINO_SPI_VERSION);
       throw new TrinoException(GravitinoErrorCode.GRAVITINO_UNSUPPORTED_TRINO_VERSION, errmsg);
     }
 
     isCoordinator = context.getNodeManager().getCurrentNode().isCoordinator();
   }
 
+  private void checkSupportCatalogNameWithMetalake(
+      ConnectorContext context, GravitinoConfig config) {
+    if (!config.simplifyCatalogNames()) {
+      int version = Integer.parseInt(context.getSpiVersion());
+      if (version < MIN_SUPPORT_CATALOG_NAME_WITH_METALAKE_TRINO_SPI_VERSION) {
+        String errmsg =
+            String.format(
+                "The Trino-%s version is not support catalog name with metalake, The minimal required version is Trino-%d",
```

Comment:
Trino-%s does not support the catalog name with dots, the minimum version required is xxxx.